crates/service_utils/src/service/types.rs

```diff
@@ -114,9 +114,7 @@ impl AppExecutionNamespace {
             scope,
         ) {
             (false, _, _, _) => Ok(AppExecutionNamespace("cac_v1".to_string())),
-            (true, _, Some(t), Some(s)) => {
-                Ok(AppExecutionNamespace(format!("{}_{}", t.as_str(), s)))
-            }
+            (true, _, Some(t), Some(_)) => Ok(AppExecutionNamespace(t.0)),
```

Comment:
can we please remove the `app_state.app_env` from the match, not being used at all
also, if possible can we simplify it ?